src/profile/request/extra_data/third_party/issuing.attributes.js

```diff
@@ -0,0 +1,28 @@
+const { Validation } = require('../../../../util');
+const SandboxDefinition = require('./definition');
+
+class SandboxIssuingAttributes {
+  /**
+   * @param {YotiDate} expiryDate
+   * @param {SandboxDefinition[]} definitions
+   */
+  constructor(expiryDate, definitions) {
+    Validation.isYotiDate(expiryDate, 'expiryDate');
+    this.expiryDate = expiryDate;
+
+    Validation.isArrayOfType(definitions, SandboxDefinition, 'definitions');
+    this.definitions = definitions;
+  }
+
+  /**
+   * @returns {Object}
+   */
+  toJSON() {
+    return {
+      expiry_date: this.expiryDate.getMicrosecondTimestamp(),
```

Comment:
This should be changed to use a millisecond timestamp inline with format expected by dynamic QR service